frontend/app/helpers/angular-rx-utils.ts

```diff
@@ -42,10 +42,10 @@ export function asyncTest<T>(done: (error?: any) => void, fn: (value: T) => any)
 }
 
 export function scopeDestroyed$(scope: IScope): Observable<IScope> {
-  return Observable.create((s: any) => {
+  return Observable.create((s:Observer<IScope>) => {
     scope.$on("$destroy", () => {
-      s.onNext(scope);
-      s.onCompleted();
+      s.next(scope);
```

Comment:
/cc @romanroe This caused errors when moving between (UI router) states due to the `scopedDestroyed$` raising errors.